asciidoc/courses/llm-chatbot-typescript/modules/1-project-setup/lessons/2-setup/lesson.adoc

```diff
@@ -54,17 +55,51 @@ git clone https://github.com/neo4j-graphacademy/llm-chatbot-typescript
 
 === Installing Dependencies
 
-You can review the dependenies required to run the project in the link:{repository-blob}/main/package.json[`package.json`] file in the project root.
+You can review the dependencies required to run the project in the link:{repository-blob}/main/package.json[`package.json`] file in the project root.
 
-Too install the dependencies, you can run the `npm install` command.
+To install the dependencies, you can run the `npm install` command.
 
 [source,sh]
 .Install Depencencies
+cd llm-chatbot-typescript
 npm install --include=dev
 
 
-=== Starting the App
+=== Setting Environment Variables
+
+To connect to Neo4j, you will need to set your Neo4j Sandbox credentials as environment variables.
+
+To set environment variables in a Next.js application, you can create a `.env.local` file.
```

Comment:
Perhaps simplify and just include a .env.local

Reply:
Including a `.env.example` is an established pattern. I'd rather be explicit about setting the credentials correctly than causing confusion by including the wrong ones.